modules/object-cache/health/load.php

```diff
@@ -0,0 +1,244 @@
+<?php
+/**
+ * Module Name: Health Check
```

Comment:
I've renamed the module to `health-check` and updated the module name in https://github.com/WordPress/performance/pull/111/commits/2e3816950af4a4a1ddea778ee390c38ca51cdb3c.

Did you mean the slug should be `persistent-object-cache-health-check`?

Reply:
>Overall I'm still unsure whether this should be in the Object cache focus or the Site Health focus. IMO the latter is more suitable, since this doesn't improve anything around object caching on the WordPress site,

At this point I don't think we'll have any object caching modules, since all improvements are so far have taken place in core itself. I'll see if there are non-core issues.

>it just informs about the state of object caching - which IMO makes it more suitable alongside other Site Health audits, in that focus area.

I agree with that, however IMO the _Object Cache_ team should be responsible for the code. Maybe we move this into _Site Health_ once we it's stable and ready for merge into 6.0? There is still #161 to tackle, likely as part of this.

Reply:
Yeah I think the module name is better now. 👍 

I don't have a strong opinion on the focus area, I agree with your point on code ownership that it makes sense to have it in the Object Cache team.